plugins/com.google.cloud.tools.eclipse.appengine.validation.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.validation
 Require-Bundle: org.junit,
- org.eclipse.equinox.registry
+ org.eclipse.equinox.registry,
+ com.google.cloud.tools.eclipse.test.dependencies
```

Comment:
The test dependencies are one of the weird corner cases where we seem to need Require-Bundle due to the conflicting versions of Mockito and Hamcrest. :-(